packages/terra-notification-dialog/src/terra-dev-site/doc/notification-dialog/NotificationDialog.1.doc.mdx

```diff
@@ -35,33 +47,97 @@ This component requires the following peer dependencies be installed in your app
 | react | ^16.8.5 |
 | react-dom | ^16.8.5 |
 | react-intl | ^2.8.0 |
+| terra-breakpoints | ^2.19.0 |
 
 <!-- AUTO-GENERATED-CONTENT:END -->
 
 ## Usage 
 ``` jsx
-import NotificationDialog, { NotificationDialogVariants } from 'terra-notification-dialog';
+import NotificationDialog from 'terra-notification-dialog';
 ```
 
 ## Component Features
 
-
  * [Cross-Browser Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support)
  * [Responsive Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#responsive-support)
  * [Mobile Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#mobile-support)
  * [LTR/RTL Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#ltr--rtl-support)
  * [Internationalization Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#internationalization-i18n-support)
  * [Localization Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#internationalization-i18n-support)
 
-## Examples 
-<CompleteNotificationDialog title='Complete Notification Dialog' />
-<CompleteNotificationDialogWithLongMessage title='Complete Notification Dialog With Long Message' />
-<NoVariantNotificationDialog title='No Variant Notification Dialog' />
-<NotificationDialogWithLongText title='Notification Dialog With Long Text' />
-<NotificationDialogOnModalManager title='Notification Dialog On Modal Manager' />
-<ContentNotificationDialog title='Notification Dialog with Content' />
-<RejectFirstNotificationDialog title='Notification Dialog with Reject action first' />
-<EmphasizedRejectNotificationDialog title='Notification Dialog with Reject Action emphasized' />
+## Hazardous Notification Dialogs 
+<HazardHighVariant
+  title="Hazard-High Notification Dialog"
+/>
+<HazardMediumVariant
+  title="Hazard-Medium Notification Dialog"
+/>
+<HazardLowVariant
+  title="Hazard-Low Notification Dialog"
+/>
+
+## Non-Hazardous Notification Dialogs 
+<ErrorVariant
+  title="Error Variant Notification Dialog"
+/>
+<CustomVariant
+  title="Custom Variant Notification Dialog"
+/>
+
+## Content Layouts
+The Notification Dialog provides the following layouts for the `content` prop:
+
+### ContentLayoutAsList
```

Comment:
Do you want to pull in the props table for this component?

Reply:
Props table is below after the NofiticationDialog's Prop Table: https://github.com/cerner/terra-framework/pull/1105/files#diff-9f590b46d228f2604680eb3db3beadc8R121